docs/process.md

```diff
@@ -170,6 +170,18 @@ You will need the specific sphinx version installed, which you can do using
 need to add `~/.local/bin` to your path, if pip installs to there).
 
 
+### Building and viewing the Website locally
+
+To build the site locally for testing purposes you only need a subset of the
+`update_docs.py` command just mentioned above. Specifically:
+
+1. Run the `pip3` command from before and adjust your path (see above).
+2. Go to `site/`.
+3. Run `make html`.
+4. Run a local webserver, like `python3 -m http.server`.
```

Comment:
`/like/e.g./`

And `python3 -m http.server site/build/html`to avoid the need to change directory.